### DISCUSSION TITLE
weeklyControl

### DISCUSSION THREAD

**Discussion by KenReneris** [nikshriv/hass_navien_water_heater#26] [General]
I'd very much like to be able to programmatically turn on & off the weeklyControl value.    I've tried modifying the code but to no avail.   The iPhone app is able to make the change, but it's unclear how to format the proper request to make it happen.   Could you help?

Note I've tried commands 33554433 and 33554438 with   mode:weeklyControl &  params:[2]   as a guess by reviewing other code but it didn't take.

**Comment by tboyko** [comment]
See #50
